package.json

```diff
@@ -47,10 +47,12 @@
     "eslint-config-webpack": "^1.2.1",
     "eslint-plugin-import": "^2.2.0",
     "extract-text-webpack-plugin": "^3.0.0",
+    "fsevents": "^1.2.4",
```

Comment:
Why is this a dependency now? This should only be used on macOS

Reply:
Previously an older version of `fsevents` was present in package-lock.json - one that led to compile failures - and I wasn't sure how to get it to update besides declaring the version directly here.